app/utils/validatePhoneNumber.js

```diff
@@ -0,0 +1,26 @@
+
+/**
+ *  @module normalize/validatePhoneNumber
```

Comment:
Forgive me if I'm wrong, but did you copy this file from somewhere else? This doc comment `@module` declaration doesn't match the actual file path, and we don't even use `@module` declarations elsewhere in the codebase. In addition, the style of this file is completely unlike the style of any of the other code in this codebase, which is why I wanted to make sure I ask.

If you did copy this from elsewhere, then 1) we really need to provide attribution for it, both for legal reasons and just for ethics, and 2) we should really check the license of the code to make sure we are compliant (most licenses at least ask you to provide attribution, but they may have stronger requirements). Also, although it's less of an issue in our codebase, since we decided to give AskDarcel one of the stricter copyleft licenses, GPLv3, but in general, you should be very careful about code that you copy from elsewhere, since the license of that code may [force you to relicense your own code that you're integrating it with](https://en.wikipedia.org/wiki/Viral_license).

If you didn't copy this, then disregard everything that I wrote above, and I apologize for falsely suspecting that this code may have been copied.

Reply:
ohh I tried to write some documentation but didn't realize I did it wrong but I did copy Regex's from stackoverflow.